File: 8-kyu/rock-paper-scissors!/python/solution.py
def rps(p1, p2):
    beat = {"scissors": "rock", "paper": "scissors", "rock": "paper"}            
    if beat[p1] == p2:
        return "Player 2 won!"
    elif beat[p2] == p1:
        return "Player 1 won!"
    else: 
        return "Draw!"

File: 8-kyu/rock-paper-scissors!/elixir/solution.ex
defmodule RockPaperScissors do
  @beats_table %{
    "scissors" => "paper",
    "paper" => "rock",
    "rock" => "scissors"
  }
  
  def rps(p1, p2) when p1 == p2, do: "Draw!"
  def rps(p1, p2), do: if @beats_table[p1] == p2, do: "Player 1 won!", else: "Player 2 won!"
end